js/info.js

```diff
@@ -667,10 +682,10 @@ var info = (function () {
         }
         olfeatures.forEach(function(feature){
             if (activeAttributeValue) {
-                feature.properties[activeAttributeValue] = true;
+                feature.setProperties({'activeAttributeValue': true});
             }
             // add a key_value array with all the fields, allowing to iterate through all fields in a mustache templaye
-            feature.properties['fields_kv'] = function () {
+            var fields_kv = function () {
```

Comment:
Create new issue to change something out of issue #315  please.

Reply:
Touching on the feature structure in info.js started when parsing the GML/GetFeatureInfo which created a "custom" feature format before, treating the geometry as any other property and making it hard to identify the geometry. Using GetFeatureInfo parser of OL facilitated this in creating OL features. To me it felt cleaner to stay with OL features within JS (until passing the object to templating at least). Isn't this linked close enough to the original issue?

Reply:
ok but need test with @jeanpommier #206 issue

Reply:
To me it seems to work correctly, but further confirmation would be great. I removed some more fields from `fields_kv` and put `serialized` into geojson (see below).